tests/HeaderTest.php

```diff
@@ -149,4 +149,38 @@ public function testNormalize($header, $result): void
         self::assertSame($result, Psr7\Header::normalize([$header]));
         self::assertSame($result, Psr7\Header::normalize($header));
     }
+
+    public function splitListProvider(): array
+    {
+        return $this->normalizeProvider();
+    }
+
+    /**
+     * @dataProvider splitListProvider
```

Comment:
why not just use the `normalizeProvider`?

Reply:
Wasn't sure whether reusing simply reusing the provider for an unrelated test was good style. I've adjusted this.